#!/usr/bin/python3
def raise_exception_msg(message=""):
    """A functin that Raise a NameError exception with a message."""
    raise NameError(message)
